samples/Java/azure-servicebus/TopicClientQuickstart/readme.md

```diff
@@ -0,0 +1,44 @@
+# Topic Client Quickstart
```

Comment:
Could you add a small comment on difference between a queue and a topic-subscription?

Reply:
For that I do assume people have read at least the intro portion of the docs and I don't need to repated that here.